src/Bot.mjs

```diff
@@ -359,12 +359,11 @@ class SlackBot extends Adapter {
    * @private
    */
   close() {
-    // NOTE: not confident that @options.autoReconnect works
-    if (this.options.autoReconnect) {
+    if (this.socket.autoReconnectEnabled) {
```

Comment:
I removed the `this.options.autoReconnect` comment because I don't see that option elsewhere in the code.

Up replaced the conditional to look for the `SocketModeClient.autoReconnectEnabled` to get the nice log output.